iac-extensions/arm/src/main/java/org/sonar/iac/arm/checks/SecureValuesExposureCheck.java

```diff
@@ -77,7 +76,7 @@ private static void checkSecureParametersInNestedTemplates(ContextualResource re
   }
 
   static Stream<Expression> extractPropertyValuesFromTemplate(ContextualArray resources) {
-    // TODO: traversal of resources in nested templates can be improved after https://sonarsource.atlassian.net/browse/SONARIAC-1058
+    // TODO SONARIAC-1058: traversal of resources in nested templates can be improved after they are properly supported
```

Comment:
Basically, reformatted the comments to the same format `TODO SONARIAC-\d+` to make them easier to discover and created a couple of new ones.